docusaurus/docs/publish-a-plugin/build-automation.md

```diff
@@ -52,7 +52,46 @@ After you push the tag, you can create the same tag again.
 
 ## Download the release zip file
 
-Access the final release zip file directly from the GitHub repository release path (e.g. https://github.com/org/plugin-id/releases).
+Access the final release zip file directly from the GitHub repository release path (for example, `https://github.com/org/plugin-id/releases`).
+
+## Provenance attestation for plugin builds
+
+Provenance attestation, that is, _a feature that generating verifiable records of the build's origin and process_,  enhances the security of your plugin builds. This feature allows users to confirm that the plugin they are installing was created through your official build pipeline.
+
+Currently, this feature is available only with GitHub Actions. While we recommend using GitHub Actions with provenance attestation for improved security, you can still build and distribute plugins using other CI/CD platforms or manual methods.
```

Comment:
Probably also should mention - public repositories only 

> artifact attestations are only available for public repositories

https://docs.github.com/en/actions/security-for-github-actions/using-artifact-attestations/using-artifact-attestations-to-establish-provenance-for-builds